build/Build.cs

```diff
@@ -292,7 +312,7 @@ ModifiableFileWithRestoreContentsOnDispose UpdateMsiProductVersion()
         return new ModifiableFileWithRestoreContentsOnDispose(productWxsFilePath);
     }
 
-    void RunBuildFor(string framework, string runtimeId)
+    void RunBuildFor(string framework, string? runtimeId)
```

Comment:
Why can `runtimeId` be `null` now? How does that impact the `configuration` string below?

Reply:
Nice pickup. This was changed for an experimental approach but I forgot to revert it back.